starlite/router.py

```diff
@@ -223,32 +221,49 @@ def route_handler_method_view(
                 route_map[cast("WebSocketRoute", route).route_handler.fn.__qualname__].append(route.path)  # type: ignore
         return route_map
 
-    @staticmethod
+    RouteHandlerMap = ItemsView[
```

Comment:
you can move it to `starlite.types.composite`, although its fine to leave in place as well.